modules/dicehub/dbclient/extention.go

```diff
@@ -282,14 +282,17 @@ func (client *DBClient) DeleteExtensionVersion(name, version string) error {
 	return client.Where("name = ? and version =?", name, version).Delete(&ExtensionVersion{}).Error
 }
 
-func (client *DBClient) QueryExtensionVersions(name string, all string) ([]ExtensionVersion, error) {
+func (client *DBClient) QueryExtensionVersions(req *apistructs.ExtensionVersionQueryRequest) ([]ExtensionVersion, error) {
 	var result []ExtensionVersion
 	query := client.Model(&ExtensionVersion{}).
-		Where("name = ?", name)
+		Where("name = ?", req.Name)
 	// 不显式指定all=true,只返回public的数据
-	if all != "true" {
+	if req.All != "true" {
 		query = query.Where("public = ?", true)
 	}
+	if req.OrderByVersionDesc {
+		query = query.Order("version desc")
+	}
```

Comment:
do you need add `version asc` if `orderByVersionDesc` is false?

Reply:
if `orderByVersionDesc is false`, version is order asc
so don't need add `version asc`